### DISCUSSION TITLE
Product Variations in Loop Grid

### DISCUSSION THREAD

**Discussion by nlaz85** [elementor/elementor#25573] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

Hi guys,
I don't know about you, but I often make purchases directly from the archive page without going through the product page. For variable products, this becomes impossible with Loop Grid because I have to go to the product page to select the variation (size, color, etc.)

### Describe the solution you'd like.

It would be nice if there was a widget that allowed you to add to the loop the possibility to select variations and add them directly to the cart without having to go through the product page.

The display of the variations could also have labels instead of a dropdown (a solution that can only be implemented with third-party plugins).

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by waqarali2048** [comment]
Hi, were you able to find any solution?
